### PR TITLE
Configures woloxci

### PR DESCRIPTION
**Pull request by mdesanti**
## Summary

Configures Woloxci

### CODE REVIEW

File: .woloxci/config.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+config:
+  dockerfile: .woloxci/Dockerfile
+  project_name: rails-bootstrap
```
Comment (alanhala) [inline .woloxci/config.yml]: Shouldn't we change this name when we run the bootstrap script?

File: .woloxci/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM ruby:2.4.0
```
Comment (alanhala) [inline .woloxci/Dockerfile]: We have another dockerfile in the project. Shouldn't we remove that one? I assume the other one was used for Jenkins

File: .woloxci/config.yml (hunk 1 above)
Comment (mdesanti) [reply .woloxci/config.yml]: That should be taken care of by the bootstrap script, which searches the entire repo for the word `rails-bootstrap`

File: .woloxci/Dockerfile (hunk 2 above)
Comment (mdesanti) [reply .woloxci/Dockerfile]: You're right

**Comment by hdf1986** [comment]
@alanhala i don't agree, it's useful to have that, remember that this repo is opensource and not everyone has access to our ci. Also in case we need to build on travis we are ready to go.

**Pull request merged**